students/ZidanLuo/session02/printGrid.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+def print_grid(x,y):
+    count = 1
+    #count all the possible number of lines
+    lines = (y + 1) * x + 1
+    plus = "+"
```

Comment:
is there any poiint in defining a variable that is a single character anyway?

Maybe -- if you want to be able to change that character later in one place. But in that case, maybe give it a more generic name, like "corner" rather than "plus".